### ISSUE TITLE
[Batch] Output layout can break other people code

### ISSUE THREAD

**Issue by vasole** [vasole/pymca#490]
Should legacy output be at the same place as before?

People are used to find an IMAGES folder inside the output directory with the latest .dat and .edf files. I understand the convenience of the new layout in order to be able to use the same destination directory for several treatments, but I am not sure about the convenience to drop that folder.

**Comment by vasole** [comment]
@woutdenolf 

Would it be very difficult/time consuming to create the IMAGES folder inside the output folder when the user has selected legacy output?

**Comment by woutdenolf** [comment]
So instead of
```
map_xia00_0001_0000_0000_to_0004.h5::map_xia00_0001_0000_0000_to_0004
map_xia00_0001_0000_0000_to_0004/map_xia00_0001_0000_0000_to_0004.edf
```
you expect
```
map_xia00_0001_0000_0000_to_0004.h5::map_xia00_0001_0000_0000_to_0004
IMAGES/map_xia00_0001_0000_0000_to_0004.edf
```
Correct? For fast and slow fit? That's very simple to do: I just need to modify `OutputBuffer::outroot_localfs` (and associated unittests). I've always found this a peculiar feature though... What's the reasoning behind it?

**Comment by woutdenolf** [comment]
Btw, didn't this ever cause problems with the old `LegacyPyMcaBatchBuildOutput`? If you have two multiprocess batches running at the same time, the first one that finished tries to merge all parts. Anyway, the new `PyMcaBatchBuildOutput` has a `basename` keyword for this so it wouldn't be a problem anymore.

**Comment by vasole** [comment]
closed by #508
